dashboard.py

```diff
@@ -6,65 +6,105 @@
 import plotly.express as px
 from dash.dependencies import Input, Output
 import pandas as pd
+import requests
+from config import TOKEN
+import json
 
 
 app = dash.Dash()
 
 # df = px.data.stocks()
 
-xls = pd.ExcelFile('datasets/Modular_Link_MHA_Prototype.xlsx')
-ur_df = pd.read_excel(xls, 'UR_0')
-ur_df["modular_link_mha_prototype_linkrating"] = ur_df["modular_link_mha_prototype_linkrating"] * 4 + 1
-
-tsc_df = pd.read_excel(xls, 'TSC_4')
-tsc_df["modular_link_mha_prototype_linkrating"] = tsc_df["modular_link_mha_prototype_linkrating"] * 4 + 1
-
-df = pd.concat([ur_df, tsc_df])
-
-contextual_variables = [
-    "modular_link_mha_prototype_hadrecentactivitylast48hours",
-    "modular_link_mha_prototype_isweekend", 
-    "modular_link_mha_prototype_timeofday", 
-    "modular_link_mha_prototype_highmood", 
-    "modular_link_mha_prototype_highenergy", 
-    "modular_link_mha_prototype_studyday", 
-    "modular_link_mha_prototype_k10", 
-    "modular_link_mha_prototype_averageresponsiveness", 
-    "modular_link_mha_prototype_averagecontentratingsoverall"
-]
+mooclet_id = [315, 295]
+
+# Get mooclet policy name and corresponding contextual variable.
+mooclet = {}
+context_var = {}
+outcome_var = {}
+for id in mooclet_id:
+    mooclet_url = 'https://mooclet.canadacentral.cloudapp.azure.com/engine/api/v1/mooclet/' + \
+        str(id)
+    mresponse = requests.get(url=mooclet_url, headers={
+                             'Authorization': TOKEN})
+    name = mresponse.json()['name']
+    mooclet[id] = name
+    print(mooclet)
+    cv_url = 'https://mooclet.canadacentral.cloudapp.azure.com/engine/api/v1/policyparameters?mooclet=' + \
+        str(id) + '&policy=6'
+    dresponse = requests.get(url=cv_url, headers={
+                             'Authorization': TOKEN})
+    context_var[id] = dresponse.json(
+    )['results'][0]['parameters']['contextual_variables']  # Add Try/catch / if else for edge case handle
+    context_var[id].remove('version')
+    outcome_var[id] = dresponse.json(
+    )['results'][0]['parameters']['outcome_variable']  # Add Try/catch / if else for edge case handle
+
+
+# Read in all the data - hardcoded for now
+# 315
+mlmxls = pd.ExcelFile('datasets/Modular_Link_MHA_Prototype.xlsx')
+mlmur_df = pd.read_excel(mlmxls, 'UR_0')
```

Comment:
TODO: We need to scale up the reward value to the range of 1 to 5. But be careful about some values which are already scaled up in the dataset.